src/autotuner/genetic_search.cc

```diff
@@ -164,6 +193,7 @@ GeneticSearch::GeneticSearch(
     : population(),
       lastBestConf(confs[0]),
       kMaxPopulationSize(n),
+      kMatingPoolSize(n * 3),
```

Comment:
just curious: what is the reason for number 3? can we move the number to header file as a `static constexpr int`? :)

Reply:
Right now it is semi-arbitrary: it is the maximum number of candidates that can be used during breeding. 

I'll expose it as an option in a subsequent commit.